server/src/main/resources/db/common/V90__create_refresh_token_table.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE refresh_token
+(
+    id varchar PRIMARY KEY,
+    refresh_token varchar,
+    user_name varchar,
+    date_created timestamp,
+    revoked boolean
+);
```

Comment:
Adds a persistable refresh token. Included within this table is the `revoked` field. This is not yet implemented but could be useful in the future to revoke a refresh token if an employee no longer should have access but their browser still have an access token.